### ISSUE TITLE
[Bug Report][3.7.19] V-select not passing accessibility checks

### ISSUE THREAD

**Issue by kmmerc18** [vuetifyjs/vuetify#21164]
### Environment
**Vuetify Version:** 3.7.19
**Vue Version:** 3.5.13
**Browsers:** Chrome 134.0.0.0
**OS:** Mac OS 10.15.7

### Steps to reproduce
With a v-select open, run the axe devtool to check for accessibility errors

### Expected Behavior
There should be no errors relating to the v-select

### Actual Behavior
The underlying v-list inside the v-select is not getting an aria label, causing the issue "Aria input fields must have an accessible name"

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNpVj10LgjAUhv/K2I03pkR0I0sIoX5AQRfqxdKTDOY29iFF9N+bMzHv3j3vOOc85Rsb3aRHpZLBAc4wsdArTi3klUCIDBuqVIjh0UhhKROgfyhAAxwaOwOEOL0DP1T4MnG8NFkPwvnGagcrzvxW44syKihnD6kFo1GMokJyqWkrx3zykbUBn0HqbvpxhSc1Y7i9ZM9EF9XL4Jyk83Wzggdrh5FMiiT9U8efGO+SfbLd4Vg4zusv0ztaiw==)
